qemu/tests/chardev_add_port.py

```diff
@@ -0,0 +1,43 @@
+from virttest import error_context
+from virttest import utils_test
+from virttest import qemu_monitor
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Add existed virtio serial port and serial bus
+    1) Start guest with virtio-serial-port and virtio-serial-pci
+    2) execute":"qmp_capabilities"
+    3) Hot-plug existed port1
+    4) Hot plug existed virtio-serial-pci
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    if params['os_type'] == 'windows':
+        utils_test.qemu.windrv_check_running_verifier(
+            vm.wait_for_login(), vm, test, 'vioser', 300)
+
+    assert 'command ignored' in vm.monitor.cmd_obj({"execute": "qmp_capabilities"})['error']['desc']
+    pci_info = vm.monitor.info('pci', debug=False)
+    pci_dev_id = None
+    for dev in pci_info[0]['devices']:
+        if dev.get('class_info'):
+            if dev.get('class_info').get('class') == 1920:
+                pci_dev_id = dev.get('qdev_id')
+                p = dev
+                break
```

Comment:
What is L24~L32 ?

Reply:
You want to get the serial bus id ?

Reply:
Yes I just deleted line 31 which is not necessary

Reply:
For this case, only one virtserialport use the serial bus, i prefer:
```
    virtserial_bus_id = virtio_port.params['bus'].split('.')[0]
    virtserial_device = vm.devices.get(virtserial_bus_id)
    virtserial_device.hotplug(vm.monitor)
```